modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -1011,7 +1011,7 @@ public void testCreateSortedIndex() {
                 .tableName(TABLE_NAME)
                 .unique()
                 .columns(List.of("VAL", "ID"))
-                .collations(List.of(CatalogColumnCollation.DESC_NULLS_FIRST, CatalogColumnCollation.ASC_NULLS_LAST))
+                .collations(List.of(CatalogColumnCollation.DESC_NULLS_FIRST, ASC_NULLS_LAST))
```

Comment:
Let's revert this or convert to static on-demand import for `DESC_NULLS_FIRST`

Reply:
fix it